meshtastic/interfaces/ble/compatibility_service.py

```
@@ -572,7 +572,9 @@ def _publish_status_async(reason: str) -> bool:
             def _async_publish() -> None:
                 try:
                     _publish_status()
-                except Exception:  # noqa: BLE001 - async fallback must remain best effort
+                except (
+                    Exception
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)